lib/Makefile

```diff
@@ -22,5 +22,9 @@ install: shared
 	install $(LIBNAME) $(LIBDIR)
 	ldconfig
 
+uninstall:
+	$(RM) $(LIBDIR)/$(LIBNAME)
+	$(RM) $(LIBDIR)/$(LIBNAME).$(SOVER)
```

Comment:
You'll want to delete the `.a` file if it exists, and you'll need to run ldconfig after the uninstall is complete to ensure the symbols cache is updated.

Reply:
I've added `ldconfig`.
The `.a` isn't installed as part of `install`.

Reply:
Okay, cool.